pkg/template/registry/templateinstance/strategy.go

```diff
@@ -55,9 +55,19 @@ func (templateInstanceStrategy) PrepareForCreate(ctx apirequest.Context, obj run
 	templateInstance := obj.(*templateapi.TemplateInstance)
 
 	if templateInstance.Spec.Requester == nil {
-		user, _ := apirequest.UserFrom(ctx)
-		templateInstance.Spec.Requester = &templateapi.TemplateInstanceRequester{
-			Username: user.GetName(),
+		user, ok := apirequest.UserFrom(ctx)
+		if ok {
```

Comment:
Follow up question:  is it valid for the user name to be empty/anonymous, but have groups set?